docs/index.rst

```diff
@@ -21,6 +21,7 @@ These devices are confirmed working using the tools provided in this repository.
  - FRITZ!Box 4020
  - FRITZ!Box 4040
  - FRITZ!Box 7312
+ - FRITZ!Box 7360 (v1, v2, SL)
```

Comment:
Being picky here - The 7360SL is another model, while the v1, v2 denotes the revision.

Reply:
You mean i should split them up like this?
FRITZ!Box 7360 SL
FRITZ!Box 7360 (v1, v2)

Reply:
correct

Reply:
Done. I must admit at second look this really looks cleaner.